tests/use_cases/test_injection_use_case.py

```diff
@@ -0,0 +1,41 @@
+import mock
+
+from snipssonos.use_cases.inject_entities import InjectEntitiesUseCase
+from snipssonos.entities.artist import Artist
+from snipssonos.shared.response_object import ResponseSuccess, ResponseFailure
+from snipssonos.use_cases.request_objects import InjectEntitiesRequestFactory
+
+ARTIST_ENTITY_NAME = "snips/artist"
+
+
+@mock.patch('snipssonos.services.spotify.music_custom_service.SpotifyCustomService')
+@mock.patch('snipssonos.services.injection_service.InjectEntitiesService')
+def test_inject_entities_successful(custom_mock, injection_mock):
+    custom_mock_instance = custom_mock.return_value
+    injection_mock_instance = injection_mock.return_value
+
+    custom_mock_instance.fetch_top_artist.return_value = [Artist("uri_1", "Kendrick Lamar"),
```

Comment:
I still don't understand this as you don't mock the return value

Reply:
I do, I am assigning what I want to be returned by the `fetch_top_artist` function. Same deal as in the previous comment you did about mocking